Document/0x05i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -279,9 +281,10 @@ Android apps often make use of third party libraries. These third party librarie
 - Libraries that are not (or should not) be packed within the actual production application, such as `Mockito` used for testing and libraries like `JavaAssist` used to compile certain other libraries.
 - Libraries that are packed within the actual production application, such as `Okhttp3`.
 
-These libraries can have the following two classes of unwanted side-effects:
+These libraries can lead to unwanted side-effects:
 
 - A library can contain a vulnerability, which will make the application vulnerable. A good example are the versions of `OKHTTP` prior to 2.7.5 in which TLS chain pollution was possible to bypass SSL pinning.
+- A library can no longer be maintained or hardly be used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application through the library.
```

Comment:
Slightly re-phrased. What do you think? 

```suggestion
- A library is no longer maintained or is not widely used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application because of that library.
```